### PR TITLE
Install Helix Python packages into a venv in a separate docker stage

### PR DESCRIPTION
**Pull request by jkoritzinsky**
Install Helix python packages into a separate docker stage into a venv to avoid installing and uninstalling packages from the final image and to avoid using `--break-system-packages`.

### CODE REVIEW

**Comment by jkoritzinsky** [comment]
I've validated that installing the packages into an activated venv does not break the dotnet/runtime CI in https://github.com/dotnet/runtime/pull/105157.

**Comment by richlander** [comment]
Thanks! So much better!

File: src/alpine/3.17/helix/amd64/Dockerfile [hunk 1]
```diff
@@ -85,4 +81,9 @@ RUN /usr/sbin/adduser -D -g '' -G adm -s /bin/bash -u 1000 helixbot && \
 
 USER helixbot
 
-RUN python3 -m venv /home/helixbot/.vsts-env
+# Install Helix Dependencies
+
+ENV VIRTUAL_ENV=/home/helixbot/.vsts-env
+RUN python3 -m venv $VIRTUAL_ENV
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+COPY --from=venv /venv /home/helixbot/.vsts-env
```
Comment (mthalman) [inline src/alpine/3.17/helix/amd64/Dockerfile:89]:
```suggestion
COPY --from=venv /venv $VIRTUAL_ENV
```

**Review by mthalman — APPROVED** [review]

**Comment by mthalman** [comment]
Looks like there are issues with arm32.

**Comment by richlander** [comment]
> Looks like there are issues with arm32.

I thought that was the rule!

**Pull request merged**